storage/storage.go

```diff
@@ -46,8 +53,78 @@ type Handler interface {
 	RemoveSortedSetRange(string, string, string) error
 }
 
+const defaultHashAlgorithm = "murmur64"
+
+// If hashing algorithm is empty, use legacy key generation
+func GenerateToken(orgID, keyID, hashAlgorithm string) (string, error) {
+	if keyID == "" {
+		keyID = strings.Replace(uuid.NewV4().String(), "-", "", -1)
```

Comment:
it is not efficient to get canonical string rep of UUID and then remove dashes right away using string search/replace operation, maybe it is OK as long as we don't issue tons of tokens per sec